catalog/transformer/catalog-transformer-service-xslt/src/main/java/ddf/catalog/services/xsltlistener/AbstractXsltTransformer.java

```diff
@@ -64,13 +65,12 @@ public AbstractXsltTransformer(Bundle bundle, String xslFile) {
   public void init(final Bundle bundle, String xslFile) {
     context = bundle.getBundleContext();
     URL xsltUrl = bundle.getResource(xslFile);
-    String mimeType = (String) bundle.getHeaders().get(MIME_TYPE_HEADER_NAME);
+    String type = bundle.getHeaders().get(MIME_TYPE_HEADER_NAME);
     try {
-      init(mimeType, xsltUrl.openStream());
-    } catch (IOException ioe) {
-      throw new RuntimeException(
-          "Could not load xsl file (" + xslFile + ") from system: " + ioe.getMessage(),
-          ioe.getCause());
+      init(type, xsltUrl.openStream());
+    } catch (IOException | TransformerConfigurationException e) {
+      throw new IllegalStateException(
```

Comment:
✏️ This might read more cleanly with a format string.
```java
throw new IllegalStateException(String.format("Could not load xsl file (%s) from system: %s", xslFile, e.getMessage()), e.getCause());
```

Reply:
Changed to use suggested format.